hat/app/org/hatdex/hat/utils/NetworkRequest.scala

```diff
@@ -0,0 +1,60 @@
+package org.hatdex.hat.utils
+
+import io.dataswift.adjudicator.Types.{ Contract, ContractId }
+import play.api.libs.ws.{ WSClient, WSRequest, WSResponse }
+
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.concurrent.Future
+
+// TODO: rename to adjudicator
+// TODO: details from config
+object NetworkRequest {
```

Comment:
I love objects, however in this case im wondering if a class would be better suited ?
Each method takes a WSClient and an adjudicatorEndpoint, if this was a class those could be passed as constructor params and it would simplify the function calls